src/performance_profiles.jl

```diff
@@ -152,3 +154,49 @@ function performance_profile(
     kwargs...,
   )
 end
+
+"""
+export_performance_profile(T; solver_names = [], logscale::Bool = true, sampletol=0.0, drawtol=0.0,)
+
+Export a performance profile plot data as .csv file. Profiles data are padded with NaN to ensure .csv consistency.
+
+## Arguments
+
+* `T :: Matrix{Float64}`: each column of `T` defines the performance data for a solver (smaller is better).
+  Failures on a given problem are represented by a negative value, an infinite value, or `NaN`.
+* `filename :: String` : path to the export file.
+
+## Keyword Arguments
+
+* `solver_names :: Vector{S}` : names of the solvers
+
+Other keyword arguments are passed `performance_profile_data`.
+"""
+function export_performance_profile(
+  T::Matrix{Float64},
+  filename::String;
+  solver_names::Vector{S} = String[],
+  kwargs...
+) where {S <: AbstractString}
+  nsolvers = size(T)[2]
+
+  x_data, y_data, max_ratio = performance_profile_data(T,kwargs...)
+  max_elem = maximum(length.(x_data))
+  for i in eachindex(x_data)
+    append!(x_data[i],[NaN for i=1:max_elem-length(x_data[i])])
+    append!(y_data[i],[NaN for i=1:max_elem-length(y_data[i])])
+  end
+  x_mat = hcat(x_data...)
+  y_mat = hcat(y_data...)
+
+  isempty(solver_names) && (solver_names = ["solver_$i" for i = 1:nsolvers])
+  @show nsolvers solver_names
+
+  header = vcat([[sname*"_x",sname*"_y"] for sname in solver_names]...)
```

Comment:
Could we add this to the kwargs?

Reply:
Do you mean the header ?

Reply:
Yes! So, that a user could personalize this without modifying the CSV file.

Reply:
It's done!